lib/vyasa_web/live/source_live/chapter/index.html.heex

```diff
@@ -1,34 +1,37 @@
+<br/>
+<div id="chapter-index-container">
     <.header>
         <div class="font-dn text-4xl mb-4">
-           <%= @selected_transl.target.translit_title %> | <%= @chap_title %>
+           <%= @selected_transl.target.translit_title %> | <%= @chap.title%>
         </div>
        <div class="font-dn text-2xl mb-4">
-        Chapter <%= @chap_no%> - <%= @selected_transl.target.title %>
+        Chapter <%= @chap.no%> - <%= @selected_transl.target.title %>
         </div>
 
         <:subtitle>
             <%= @selected_transl.target.body %>
         </:subtitle>
     </.header>
 
-    <.verse_list :for={{_dom_id, verse} <- @streams.verses}>
+    <.verse_list id={dom_id} :for={{dom_id, verse} <- @streams.verses}>
```

Comment:
NB: @rtshkmr learnt that the :for is not a scope defininig syntax, it's actually just a generator fn